core/code/portal_info.js

```diff
@@ -349,7 +349,8 @@ window.getPortalHackDetails = function (d) {
   // first mod of type is fully effective, the others are only 50% effective
   var effectivenessReduction = [1, 0.5, 0.5, 0.5];
 
-  var cooldownTime = window.BASE_HACK_COOLDOWN;
+  var isFriendly = window.teamStringToId(d.team) === window.teamStringToId(window.PLAYER.team); 
```

Comment:
teamStringToId calls are redundant.

Reply:
> teamStringToId calls are redundant.
portailDetails.team = "R"

window.PLAYER.team = "RESISTANCE"

no, it's not redundant